src/main/java/com/iota/iri/conf/TestnetConfig.java

```diff
@@ -0,0 +1,161 @@
+package com.iota.iri.conf;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+
+import java.util.Objects;
+
+public class TestnetConfig extends NetConfig {
+
+    protected String coordinator = Defaults.COORDINATOR_ADDRESS;
+    protected boolean validateTestnetMilestoneSig = Defaults.VALIDATE_MILESTONE_SIG;
+    protected String snapshotFile = Defaults.SNAPSHOT_FILE;
+    //TODO should default testnet file be the same as mainnet?
+    protected String snapshotSignatureFile = Defaults.SNAPSHOT_SIG;
+    protected long snapshotTime = Defaults.SNAPSHOT_TIME;
+    protected int minimumWeightMagnitude = Defaults.MWM;
+    protected int milestoneStartIndex = Defaults.MILESTONE_START_INDEX;
+    protected int numberOfKeysInMilestone = Defaults.KEYS_IN_MILESTONE;
+    protected int transactionPacketSize = Defaults.PACKET_SIZE;
+    protected int requestHashSize = Defaults.REQUEST_HASH_SIZE;
+
+    public TestnetConfig() {
+        super();
+        dbPath = Defaults.DB_PATH;
+        dbLogPath = Defaults.DB_LOG_PATH;
+    }
+
+    @Override
+    public String getCoordinator() {
+        return coordinator;
+    }
+
+    @Parameter(names = "--testnet-coordinator", description = CooConfig.Descriptions.COORDINATOR)
+    protected void setCoordinator(String coordinator) {
+        this.coordinator = coordinator;
+    }
+
+    @Override
+    public boolean isValidateTestnetMilestoneSig() {
+        return validateTestnetMilestoneSig;
+    }
+
+    @Parameter(names = "--validate--testnet--milestone", description = CooConfig.Descriptions.VALIDATE_TESTNET_MILESTONE_SIG)
```

Comment:
Should this be `--validate-testnet-milestone` instead?

Reply:
Changed to `--testnet-no-coo-validation` due to backwards comparability issues. If you don't like the phrasing of the flag you can open an issue. However it can't be changed in the context of this PR because I don't want to break the backwards comparability.